gcloud-java-core/src/test/java/com/google/cloud/ServiceOptionsTest.java

```diff
@@ -215,6 +219,16 @@ public void testBaseEquals() {
   }
 
   @Test
+  public void testLibraryName() {
+    assertEquals(LIBRARY_NAME, OPTIONS.libraryName());
+  }
+
+  @Test
+  public void testApplicationName() {
```

Comment:
I am not sure where it would be placed in the tests.jar (under resources or at the top and if the latter would it be overridden with the manifest created by the jar) but I think it is doable. future PR sounds good to me.